modules/account-lib/src/coin/hbar/transaction.ts

```diff
@@ -237,7 +241,7 @@ export class Transaction extends BaseTransaction {
   /**
    * Returns a hash of the given proto object.
    *
-   * @param obj - The object to be hashed, it must be an proto namespace object
```

Comment:
What's this `proto` type? Can we use it to constrain the generic type `T`?

Reply:
I think it may require quite a bit of code changes unfortunately :(